app/test_main.py

```diff
@@ -1,3 +1,27 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+def test_years_negative_or_equal_zero() -> None:
+    assert min(get_human_age(-1, 0)) == 0
```

Comment:
You should check that both negative value and zero give zero in the list result. So assert it with list [0. 0], because min() can work not correct